tools/topology/topology2/include/pipelines/cavs/passthrough-playback-be.conf

```diff
@@ -32,17 +32,12 @@ Class.Pipeline."passthrough-playback-be" {
 			"index"
 		]
 
-		#
-		# mixin-playback objects instantiated within the same alsaconf node must have
-		# unique pipeline_id attribute
-		#
 		unique	"instance"
 	}
 
 	Object.Widget {
 		copier."1" {
-			type dai_in
-			node_type $HDA_LINK_OUTPUT_CLASS
```

Comment:
if you remove the default node_type, I think we should make sure it is set in all top-level conf files first. I think we avoid setting it for HDA

Reply:
I don't know how to progress. This is wrong in general. It should never have been set this way, if there is an attribute that is not valid by default across all uses, then it cannot be initialized by default in the base class.